flash/video/classification/data.py

```diff
@@ -566,6 +569,131 @@ def from_data_frame(
+        :ref:`customizing transforms guide <customizing_transforms>`.
+
+        Args:
+            input_field: The field (key name) in ``dict`` containing the video tensors.
+            target_field: The field (key name) in the ``dict`` containing the targets.
+            train_data: The ``dict`` containing tensors in ``input_field`` key and targets in
+                ``target_fields`` key to use when training.
+            val_data: The ``dict`` containing tensors in ``input_field`` key and targets in
+                ``target_fields`` key to use when validating.
+            test_data: The ``dict`` containing tensors in ``input_field`` key and targets in
+                ``target_fields`` key to use when testing.
+            target_formatter: Optionally provide a :class:`~flash.core.data.utilities.classification.TargetFormatter` to
+                control how targets are handled. See :ref:`formatting_classification_targets` for more details.
+            video_sampler: Sampler for the internal video container. This defines the order tensors are used and,
+                if necessary, the distributed split.
+            input_cls: The :class:`~flash.core.data.io.input.Input` type to use for loading the data.
+            predict_input_cls: The :class:`~flash.core.data.io.input.Input` type to use for loading the prediction data.
+            transform: The :class:`~flash.core.data.io.input_transform.InputTransform` type to use.
+            transform_kwargs: Dict of keyword arguments to be provided when instantiating the transforms.
+            data_module_kwargs: Additional keyword arguments to provide to the
+                :class:`~flash.core.data.data_module.DataModule` constructor.
+
+        Returns:
+            The constructed :class:`~flash.video.classification.data.VideoClassificationData`.
+
+        Examples
+        ________
+
+        .. doctest::
+
+            >>> import torch
+            >>> from flash import Trainer
+            >>> from flash.video import VideoClassifier, VideoClassificationData
+            >>> input_video = torch.randint(low=0, high=255, size=(3, 10, 10, 10), dtype=torch.uint8, device="cpu")
+            >>> train_data = {
+            ...     "data": torch.stack(
+            ...         (
+            ...             input_video,
+            ...             input_video,
+            ...         )
+            ...     ),  # 2 videos (each video: 10 frames)
+            ...     "targets": ["fruit", "vegetable"],  # Labels corresponding to each video
+            ... }
+            >>> predict_data = {
+            ...     "data": torch.stack((input_video,)),
+            ... }
+            >>> datamodule = VideoClassificationData.from_tensors(
+            ...     input_field="data",
+            ...     target_field="targets",
+            ...     train_data=train_data,
+            ...     predict_data=predict_data,
+            ...     batch_size=1,
+            ... )
```

Comment:
I think it would be better to make the API consistent with what we have for image classification. E.g. like this:
```
            >>> datamodule = VideoClassificationData.from_tensors(
            ...     train_data=[input_video, input_video, input_video],
            ...     train_targets=[1, 2, 3],
            ...     predict_data=predict_data,
            ...     batch_size=1,
            ... )
```